server/src/load_data.py

```diff
@@ -25,6 +25,7 @@
 redis_conn = redis.from_url(config.REDIS_URL)
 
 
+# TODO: This function is never used.
```

Comment:
create_index and create_flat_index cand be deleted

Reply:
Deleted. Note that I also deleted an invocation of `create_flat_index`. Hopefully, it's correct.